lms/djangoapps/certificates/views/webview.py

```diff
@@ -252,7 +252,7 @@ def _update_social_context(request, context, course, user, user_certificate, pla
     """
     share_settings = getattr(settings, 'SOCIAL_SHARING_SETTINGS', {})
     context['facebook_share_enabled'] = share_settings.get('CERTIFICATE_FACEBOOK', False)
-    context['facebook_app_id'] = getattr(settings, "FACEBOOK_APP_ID", None)
+    context['facebook_app_id'] = microsite.get_value("FACEBOOK_APP_ID", getattr(settings, "FACEBOOK_APP_ID", None))
```

Comment:
Well secret and version are not required for Facebook shares thats why we are not using it.